cardano-node/src/Cardano/Node/TUI/LiveView.hs

```diff
@@ -118,7 +105,7 @@ instance NoUnexpectedThunks a => NoUnexpectedThunks (LiveViewBackend blk a) wher
     return result
 
 type LiveViewMVar blk a = MVar (LiveViewState blk a)
-newtype LiveViewBackend blk a = LiveViewBackend { getbe :: LiveViewMVar blk a }
+data LiveViewBackend blk a = LiveViewBackend { getbe :: LiveViewMVar blk a }
```

Comment:
Why?

Reply:
I've experimented with additional fields, just forgot to return `newtype`. Fixed.